src/snowflake/snowpark/modin/plugin/_internal/window_utils.py

```diff
@@ -78,7 +83,7 @@ def check_and_raise_error_rolling_window_supported_by_snowflake(
     step = rolling_kwargs.get("step")
 
     # Raise not implemented error for unsupported params
-    if not isinstance(window, int):
+    if not isinstance(window, (int, str)):
         ErrorMessage.method_not_implemented_error(
             name="Non-integer window", class_="Rolling"
         )
```

Comment:
```suggestion
        ErrorMessage.not_implemented(
        	"Snowpark pandas does not yet support rolling or window " +
        	"methods with windows that are not strings or integers."
        )
```

right now the error message says, `NotImplementedError: Snowpark pandas does not yet support the method Rolling.Non-integer window`

Reply:
Good catch, will fix